src/ApiPort.VisualStudio/Analyze/ApiPortVsAnalyzer.cs

```diff
@@ -44,7 +44,7 @@ public ApiPortVsAnalyzer(ApiPortClient client, OptionsViewModel optionsViewModel
             var paths = assemblyPaths.Select(t => new AssemblyFile(t));
             var dependencyInfo = _dependencyFinder.FindDependencies(paths, _reporter);
 
-            var analysisOptions = await GetApiPortOptions(assemblyPaths, "json");
+            var analysisOptions = await GetApiPortOptions(assemblyPaths, new[] { "json" });
```

Comment:
This step is used to generate line numbers. I'll consolidate it with the previous call, but in a later PR